ui/page/wallet_settings_page.go

```diff
@@ -245,7 +245,7 @@ func (pg *WalletSettingsPage) Handle() {
 							}
 							pm.Dismiss()
 							pg.Toast.Notify("Wallet removed")
-							pg.PopFragment()
+							pg.ChangeFragment(NewWalletPage(pg.Load))
```

Comment:
This will be adding a new wallet page to the stack, PopFragment should be used to go back to the previous wallet page.